Gemfile.lock

```diff
@@ -98,7 +98,7 @@ GEM
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
     active_model_serializers (0.9.5)
       activemodel (>= 3.2)
-    activeadmin (1.0.0.pre4)
+    activeadmin (1.0.0.pre2)
```

Comment:
Needed to downgrade to make `bundle update` work. Researches brought me to:
https://github.com/activeadmin/activeadmin/issues/3926#issuecomment-233651735